can/interfaces/pcan/pcan.py

```diff
@@ -482,7 +490,7 @@ def _recv_internal(
         end_time = time.time() + timeout if timeout is not None else None
 
         while True:
-            if self.fd:
+            if self.protocol == CanProtocol.CAN_FD:
```

Comment:
```suggestion
            if self._can_protocol is CanProtocol.CAN_FD:
```
properties are rather slow, and enums are compared with `is`